lighthouse-core/gather/gather-runner.js

```diff
@@ -576,7 +577,7 @@ class GatherRunner {
         Object.assign(artifacts, passResults.artifacts);
 
         // If we encountered a pageLoadError, don't try to keep loading the page in future passes.
-        if (passResults.pageLoadError) {
+        if (passResults.pageLoadError && passConfig.loadFailureMode === 'fatal') {
```

Comment:
do we do anything with the 'warn' case?

where does the warning end up?

Reply:
message still goes in the toplevel run warnings, and all the gatherers still get errored out, it just doesn't cause the run to halt and a non-zero exit code